packages/docs/src/demos/fetch/AddonOnNode.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, waitFor, screen } from '@testing-library/react';
+import { composeStories } from '@storybook/testing-react';
```

Comment:
Is there a way to run a story in Node.js without introducing a new dependency?

I'm not against `@storybook/testing-react` in any way, my point is to keep the testing setup to the bare minimum. That always pays off. If we could find a way to run a story in Node.js directly, that'd be great.